C:\Python26>python.exe tracert.py 193.145.222.100
WARNING: No route found for IPv6 destination :: (no default route?)
1       108 ms  108 ms  110 ms  181.166.144.1
2       *       *       *       Request timed out.
3       *       *       *       Request timed out.
4       *       *       *       Request timed out.
5       108 ms  109 ms  109 ms  200.89.165.173
6       108 ms  108 ms  109 ms  200.89.165.130
7       109 ms  110 ms  108 ms  200.89.165.222
8       108 ms  108 ms  108 ms  200.63.151.57
9       109 ms  108 ms  108 ms  200.51.208.62
10      108 ms  *       *       84.16.9.233
11      219 ms  218 ms  219 ms  94.142.123.26
12      217 ms  217 ms  218 ms  94.142.126.177
13      217 ms  219 ms  218 ms  213.140.52.34
14      218 ms  219 ms  217 ms  154.54.7.157
15      218 ms  217 ms  217 ms  154.54.42.26
16      327 ms  327 ms  326 ms  154.54.46.102
17      326 ms  327 ms  328 ms  130.117.2.250
18      328 ms  327 ms  327 ms  130.117.3.118
19      328 ms  328 ms  328 ms  149.11.68.50
20      326 ms  326 ms  327 ms  130.206.245.2
21      327 ms  328 ms  326 ms  130.206.214.142
22      *       *       *       Request timed out.
23      328 ms  326 ms  327 ms  193.145.222.100

Trace complete.